src/llm/apis/openai_completions.cpp

```diff
@@ -166,7 +161,25 @@ static absl::Status downloadImage(const char* url, std::string& image, const int
     return absl::OkStatus();
 }
 
-absl::Status OpenAIChatCompletionsHandler::parseMessages() {
+ov::Tensor loadImageStbiFromMemory(const std::string& imageBytes) {
+    int x = 0, y = 0, channelsInFile = 0;
+    constexpr int desiredChannels = 3;
+    unsigned char* image = stbi_load_from_memory(
+        (const unsigned char*)imageBytes.data(), imageBytes.size(),
+        &x, &y, &channelsInFile, desiredChannels);
+    return loadImageStbi(image, x, y, desiredChannels);
+}
+
+ov::Tensor loadImageStbiFromFile(char const* filename) {
+    int x = 0, y = 0, channelsInFile = 0;
+    constexpr int desiredChannels = 3;
+    unsigned char* image = stbi_load(
+        filename,
+        &x, &y, &channelsInFile, desiredChannels);
+    return loadImageStbi(image, x, y, desiredChannels);
```

Comment:
We use raw pointers: `filename` and `image`. Are you sure they are both valid and we don't need to do any checks here?